src/applications/disability-benefits/all-claims/pages/pastEmploymentFormIntro.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
```

Comment:
remove this import since should have your content in a `src/applications/disability-benefits/all-claims/content/` file

Reply:
moved to content/